### ISSUE TITLE
Imported PDF is shown with inverted colours

### ISSUE THREAD

**Issue by pzitelli84** [OpenBoard-org/OpenBoard#319]
Hi, when I import a PDF file (created in LaTeX or saved in Impress from LibreOffice) the colours appear to be inverted: the white background is shown in black and the black characters are shown in white. The other colours suffer the same effect for objects created in Impress, for example, red is shown yellow. The images added to the original file (in LaTeX or Impress) are shown with the correct color. It seems it just affects the background, text and objects generated in Impress.

I'm using Fedora Linux 31. I also tried with PDFs created in Windows and the problem is the same.

Please let me know if you need any other information.

**Comment by kaamui** [comment]
Hello,

Can you provide a pdf reproducing the issue ? Thanks.

**Comment by pablozite** [comment]
Hi! I'm attached you will find one of the pdf files that shows the problem described (it was created in Impress by LibreOffice, equations made in LaTeX and pasted as images). I'm also attaching a screenshot of the video produced.

Funny thing: A couple days ago I upgraded to Fedora 32 and the package for OpenBoard isn't ready yet, so I had to install it using flatpak. Guess what? The pdf files are imported correctly and no inverted colours. Any way, maybe this issue can help other people.

Thanks in advance and great work!!!
[Principios_de_conservacion.pdf](https://github.com/OpenBoard-org/OpenBoard/files/4584632/Principios_de_conservacion.pdf)
![Screenshot_20200505_235006](https://user-images.githubusercontent.com/8440605/81134463-38f27180-8f2b-11ea-88d9-2be346e3a875.png)
